python/ray/util/iter.py

```diff
@@ -198,6 +198,37 @@ def for_each(self, fn: Callable[[T], U]) -> "ParallelIterator[U]":
         return self._with_transform(lambda local_it: local_it.for_each(fn),
                                     ".for_each()")
 
+    def for_each_concur(self,
+                        fn: Callable[[T], U],
+                        max_concur=2,
+                        resources=None) -> "ParallelIterator[U]":
+        """Remotely apply fn to each item in this iterator, at most max_concur at a
+        time.
+
+        A performance note: This function maintains its own internal buffer. If
+        `async_queue_depth` is `n` and max_concur is `k` then the total number
+        of buffered objects could be up to `n + k - 1`
+
+
+
+        Args:
+            fn (func): function to apply to each item.
+            max_concur (int): the max number of concurrent calls to fn
+
+        Examples:
+            >>> next(from_range(4).for_each(lambda x: x * 2).gather_sync())
+            ... [0, 2, 4, 8]
```

Comment:
This doesn't actually show me how to use `for_each_concur` right?

BTW, can you instead use `.. code-block:: python`?

Reply:
I feel like for the sake of consistency, it would be better to switch all the documentation in the file over at once. It doesn't affect the rendered html right?